build-docker-image/README.md

```diff
@@ -7,6 +7,14 @@ It creates 2 tags:
 - Commit SHA to identify this build uniquely
 - Branch name to allow reusing the cached layers from the registry
 
+If caching is enabled via reuse-cache, the caching used depends on the setting of max-cache
+- max-cache = false, (default) then it will use inline caching (min level). Cache of the final stage only is exported with the image,
+  see https://docs.docker.com/build/cache/backends/inline/ and it will use the docker build driver https://docs.docker.com/build/builders/drivers/docker/
+- max-cache = true, then it will use registry caching (max level). Cache for all stages is exported separate to the image,
+  see https://docs.docker.com/build/cache/backends/registry/ and it will use the docker-container build driver https://docs.docker.com/build/builders/drivers/docker-container/
+
+max-cache=true is preferred for multi-stage builds.
+
```

Comment:
As discussed, add how to see cache hit in build summary

Reply:
added to readme